docs/revsets.md

```diff
@@ -70,6 +70,38 @@ only symbols.
 You can use parentheses to control evaluation order, such as `(x & y) | z` or
 `x & (y | z)`.
 
+### The `all:` prefix
```

Comment:
nit: I use "modifier" instead of "prefix" in the grammar and error message because I needed some terminology for that.

Reply:
Thanks! I'll adjust it. 

I'm also wondering whether "prefix" is a better name, but that's beyond the scope of this PR.